xrpl-std/src/core/ledger_objects/traits.rs

```diff
@@ -378,3 +379,133 @@ pub trait EscrowFields: LedgerObjectCommonFields {
         }
     }
 }
+
+/// Trait providing access to fields specific to AccountRoot objects in any ledger.
+///
+/// This trait extends `LedgerObjectCommonFields` and provides methods to access
+/// fields that are specific to Escrow objects in any ledger, not just the current one.
+/// Each method requires a register number to identify which ledger object to access.
+pub trait AccountFields: LedgerObjectCommonFields {
+    /// The identifying address of the account.
+    fn get_account(&self, register_num: i32) -> Result<AccountID> {
+        ledger_object::get_account_id_field(register_num, sfield::Account)
+    }
+
+    /// AccountTxnID field for the account.
+    fn account_txn_id(&self, register_num: i32) -> Result<Hash256> {
```

Comment:
This and other optional fields should return like `Result<Option<Hash256>>`, similar to `fn get_finish_function(&self, register_num: i32) -> Result<Option<Blob>>` in the same file.

Reply:
Great catch! Fixed via 6efb3f46e2b3863a0402efc7c4f45c3ca1015600